test/next_ls/commands/to_pipe_test.exs

```diff
@@ -0,0 +1,153 @@
+defmodule NextLS.Commands.ToPipeTest do
+  use ExUnit.Case, async: true
+
+  alias GenLSP.Structures.TextEdit
+  alias GenLSP.Structures.WorkspaceEdit
+  alias NextLS.Commands.ToPipe
+
+  @moduletag :tmp_dir
+  @parse_error_code -32_700
```

Comment:
This reminds me to add test cases when I get bad args and when the code doesn't compile